### ISSUE TITLE
feature request: detect invalid reference to a workflow file

### ISSUE THREAD

**Issue by jerguslejko** [rhysd/actionlint#391]
I would like to get a lint failure, when I use incorrect reference to a workflow file in the repo.

example:

```yaml
  # ...

   - name: Deploy
     uses: ./.github/actions/whoops-I-put-wrong-path-here
     with:
       arg1: value1
```

**Comment by jerguslejko** [comment]
ah, I just read this in the docs

```
Note that actionlint does not report any error when a directory for a local action does not exist in the repository because it is a common case where the action is managed in a separate repository and the action directory is cloned at running the workflow. (See https://github.com/rhysd/actionlint/issues/25 and https://github.com/rhysd/actionlint/issues/40 for more details).
```

is there a way to run actionlint in a stricter mode, where it only considers code that is currently checked out?

**Comment by jerguslejko** [comment]
no solution, just closing the issue
